readme.md

```diff
@@ -96,6 +96,7 @@
   - [Yassine Bennkhay](https://yassine-bennkhay.tech/) - Flutter, Dart,...
   - [Yasser Tahiri](https://blog.yezz.me/) - Security, Python, APIs, ....
   - [Mouhcine Toumi](https://blog.mouhcinetoumi.com/) - Data Engineering, branding, self dev...
+  - [Haytam Bakouane](http://hbakouane.dev/) - Laravel/Vue developer
```

Comment:
Just make it `https` here and we are good to go

Reply:
Done